ui-v2/app/styles/components/main-nav-horizontal/layout.scss

```diff
@@ -0,0 +1,135 @@
+%main-nav-horizontal > ul > li,
+%main-nav-horizontal-drop-nav a {
+  position: relative;
+}
+%main-nav-horizontal [type='checkbox'] ~ div {
+  display: none;
+}
+%main-nav-horizontal [type='checkbox']:checked ~ div {
+  display: block;
+}
+%main-nav-horizontal-action {
+  display: block;
+  padding: 3px 12px;
+  white-space: nowrap;
+}
+%main-nav-horizontal-drop-nav {
+  z-index: 400;
+  /* TODO: We should probably make menu-panel default to left hand side*/
+  left: 0;
+  right: auto;
+}
+%main-nav-horizontal-drop-nav header {
+  padding: 0.2rem 1rem;
+  padding-top: 0.7rem;
+}
+%main-nav-horizontal-drop-nav div {
+  padding: 10px;
+  padding-left: 36px;
+}
+%main-nav-horizontal-drop-nav .manage-namespaces {
+  margin-top: 4px;
+  padding-top: 4px;
+}
+%main-nav-horizontal-drop-nav div::before {
+  position: absolute;
+  left: 15px;
+  top: calc(10px + 0.1em);
+}
+%main-nav-horizontal-drop-nav div {
+  max-width: fit-content;
+}
+@supports not (max-width: fit-content) {
```

Comment:
so fancy - we should do that whole "officially don't support IE 11" thing

Reply:
Hehe, yeah I think we chatted through this a bit a while back, this was a fun one to do. I think it still looks ok if the browser doesn't support any of this, its just not perfect.

Amongst the team we are kinda 'this is ok' right now, but yeah I'd definitely be up for officially saying something somewhere.